portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,7 +43,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     comments.add(text);
 
-    Entity taskEntity = new Entity("Entry");
+    Entity taskEntity = new Entity("pastComments");
```

Comment:
Why "pastComments"? why not just "Comments" or "UserComments?

Reply:
Changed to UserComments